ert3/storage/_storage.py

```diff
@@ -356,12 +364,22 @@ def _get_numerical_data(
             f"Cannot get {record_name} data, no experiment named: {experiment_name}"
         )
 
+    ## TODO: hack ?
+    split = record_name.split(".")
+    if len(split) == 2 and split[1] == "blob":
+        record_name = split[0]
```

Comment:
Is this needed?

Reply:
"blob" is used as a special index for blob records. Without it I am not able to get the record metadata.

Reply:
Yes, I see. Is it possible to store the metadata without the index? Then we would avoid this special name. Or does that require many changes elsewhere?

Reply:
I have removed the 'blob' special index. Blob records now have no index